draft-ietf-teep-protocol.md

```diff
@@ -1779,6 +1779,23 @@ TEEP Broker
   Agents are protected against such downgrade attacks based on
   features offered by the manifest itself.
 
+Replay Protection
+: TEEP supports replay protection as follows.
+  The transport protocol under the TEEP protocol might provide replay
+  protection, but may be terminated in the TEEP Broker which is not trusted
+  by the TEEP and so the TEEP protocol does replay protection itself.
+  If attestation of the TAM is used, the attestation freshness mechanism
+  provides replay protection for attested QueryRequest messages.
+  If non-attested QueryRequest messages are replayed, the TEEP Agent will generate
+  QueryResponse or Error messages, but the REE can already conduct Denial of Service
+  attacks against the TEE and/or the TAM even without the TEEP protocol.
+  QueryResponse messages have replay protection via attestation freshness mechanism or (if
+  attestation is not used) the token field in the message.
```

Comment:
How about reorder it as follows?

QueryResponse messages have replay protection via attestation freshness mechanism or the token field in the message if attestation is not used.

Reply:
Updated